controllers/customer_controller_test.go

```diff
@@ -83,6 +95,7 @@ var _ = Describe("Customer Controller", func() {
 	Describe("Delete Jira Service Desk customer", func() {
 		Context("With valid Customer AccountId", func() {
 			It("should delete the customer", func() {
+
 				_ = cUtil.CreateCustomer(mockData.SampleCustomer, ns)
```

Comment:
Customer should be declared outside of this block

Reply:
I changed nothing in this code block